source/tools/TestHarnessParameters.h

```diff
@@ -76,6 +76,13 @@ class TestHarnessParameters : public ITestHarness {
         mRunning = true;
         mTestThread = std::thread(&TestHarnessParameters::runCompleteTest, this,
                 sampleRate, framesPerBurst, numSeconds);
+        UtilClampController myUtil;
+        if (myUtil.isUtilClampSupported()) {
+            myUtil.setUtilClampMin(1024);
+            printf("Clamping value set on %d\n", myUtil.getUtilClampMin());
+        } else {
+            printf("Clamping value not supported\n");
+        }
```

Comment:
I removed that because it seemed to be boosting a thread that was not the audio thread. That cause the whole CPU cluster to get a frequency boost that did not go down. That could impact power. It did not seem necessary.

Does isUtilClampSupported() mess with the current UtilClamp setting? Querying should have no effect on the system.

Reply:
Only the first query, which is performed before setting the current utilClamp, should set sched_policy with SCHED_FIFO (by default is SCHED_NORMAL) and utilClampMin to 0 and then return to SCHED_NORMAL (utilCamp is not available). Honestly I'm not convinced of the implementation either. We should see what happens on a device that doesn't support the utilClamp. I'm open to suggestions

Reply:
isUtilClampSupported() may only be valid for the SCHED_FIFO thread. Maybe we could use a per thread variable to cache the result. This can be a future PR.